src/store/FilterStore/FilterStore.ts

```diff
@@ -0,0 +1,53 @@
+import rootStore from '@store/RootStore/instance';
+import { ILocalStore } from '@utils/useLocalStore';
+import {
+  action,
+  computed,
+  IReactionDisposer,
+  makeObservable,
+  observable,
+  reaction,
+} from 'mobx';
+
+type PrivateFields = '_categoryId' | '_optionsIsVisible';
+
+export default class FilterStore implements ILocalStore {
+  private _categoryId: number = 0;
```

Comment:
может лучше , если не выбрана категория, то null
а то вдруг будет категория 0
```suggestion
  private _categoryId: number  | null = null;
```

Reply:
fix